### ISSUE TITLE
[Feature]: Jaeger SPM - Filter metrics on page by tag

### ISSUE THREAD

**Issue by serban-nicusor-toptal** [jaegertracing/jaeger#5438]
### Requirement

Hello!

Is there a feature planned for the SPM page to allow filtering by tags? 

We would love to see data per environment but as of now, there's no option to do it in the UI, thus forcing us to have two separate instances to see the different data.

Thanks!

### Problem

Having to do duplicate instances for filtering data in the UI.

### Proposal

Implement a filtering method based on tags for the SPM ui.

### Open questions

Is this planned?

**Comment by yurishkuro** [comment]
sounds useful. Please provide more details. How are you doing it now with two instances?

**Comment by serban-nicusor-toptal** [comment]
Hey, devs want in some cases to watch the SPM metrics of PROD.
Thus one method of doing this is to just have 2 separate Jaegers. One that ingests lower-envs data and one for prod only.

**Comment by serban-nicusor-toptal** [comment]
Hello! Any luck with this?

**Comment by serban-nicusor-toptal** [comment]
Hello! Any luck with this?

**Comment by varshith257** [comment]
Hey @serban-nicusor-toptal, Jaeger v2 SPM Project has been added to the LFM Mentorship Term 2. That mentee could help you implement this or any other contributor soon(maybe me). 

Currently, most of the contributors are focusing on other issues related to the v2 roadmap and cleaning the codebase of deprecated components

**Comment by yurishkuro** [comment]
> That mentee could help you implement this

this is not accurate. Mentees work on specific projects, we can't make any promises about additional tickets. This ticket is open for volunteers.

**Comment by varshith257** [comment]
> > That mentee could help you implement this
> 
> this is not accurate. Mentees work on specific projects, we can't make any promises about additional tickets. This ticket is open for volunteers.

I agree with it. So I mentioned a mentee or any other contributor. As that mentee could be very well versed with SPM once the LFX Project is done maybe that mentee could complete it faster. So, I mentioned it   :)

**Comment by varshith257** [comment]
@serban-nicusor-toptal Couldn't you give a try?

**Comment by serban-nicusor-toptal** [comment]
Hey, I'm a bit busy with a migration but I'll take a look over a weekend.

**Comment by pipiland2612** [comment]
Hi @yurishkuro, do you think this feature worth considering? If so, is it better if I open another issue that feature the same thing but easier to get track of progress by tick box. Anyways, I think I can take this up.
